aws/resource_aws_iam_saml_provider.go

```diff
@@ -55,7 +55,7 @@ func resourceAwsIamSamlProviderCreate(d *schema.ResourceData, meta interface{})
 
 	out, err := conn.CreateSAMLProvider(input)
 	if err != nil {
-		return err
+		return fmt.Errorf("[ERROR] failed to create IAM SAML Provider: %w", err)
```

Comment:
I'm guessing some legacy resources have "[ERROR]" in the actual error messaging but we should exclude it since its unrelated to log levels here.

```suggestion
		return fmt.Errorf("error creating IAM SAML Provider: %w", err)
```